int.ts

```diff
@@ -99,7 +99,9 @@ module J2ME {
    *             +--------------------------------+
    *             | Caller FP                      |
    *             +--------------------------------+
-   *             | Callee Method Info | Marker    |
+   *             | Frame Type                     |
```

Comment:
> If we're storing this in a separate slot now we should update the FrameType constants to just use low numbers. It's easier to debug and produces faster switch statements when the numbers are dense.

Indeed. I had considered combining it with a different value, like the frame pointer, which stays low. But it seemed like a premature optimization. And it's simple to do that later. I've updated the constants to use low numbers in c53c5ef.